CountlyConnectionManager.m

```diff
@@ -169,14 +174,22 @@ - (void)proceedOnQueue
         return;
     }
 
-    [CountlyCommon.sharedInstance startBackgroundTask];
-
     NSString* queryString = firstItemInQueue;
+    NSString* endPoint = kCountlyEndpointI;
+    NSString *overrideEndPoint = [self getOverrideTagEndPoint:firstItemInQueue];
```

Comment:
use "queryString"

Reply:
lets refactor this into a function that returns a string array. first item the new query string, second item the extracted endpoint override.